service/src/test/java/bio/terra/externalcreds/pact/VerifyServicePacts.java

```diff
@@ -48,9 +51,24 @@
     properties = {"server.port=8080"},
     classes = ExternalCredsWebApplication.class)
 @ActiveProfiles({"test", "human-readable-logging"})
-@au.com.dius.pact.provider.junitsupport.Provider("ecm")
+@au.com.dius.pact.provider.junitsupport.Provider("externalcreds")
 @PactBroker
 public class VerifyServicePacts {
+  private static final String CONSUMER_BRANCH = System.getenv("CONSUMER_BRANCH");
+  @PactBrokerConsumerVersionSelectors
+  public static SelectorBuilder consumerVersionSelectors() {
+    // The following match condition basically says
+    // If verification is triggered by Pact Broker webhook due to consumer pact change, verify only
+    // the changed pact.
+    // Otherwise, this is a PR, verify all consumer pacts in Pact Broker marked with a deployment
+    // tag (e.g. dev, alpha).
+    if (StringUtils.isBlank(CONSUMER_BRANCH)) {
+      return new SelectorBuilder().mainBranch().deployedOrReleased();
```

Comment:
thanks @ichengchang ! this worked when I ran the verification code locally, with hardcoded broker credentials. I think I'm going to get the CBAS side merged in and then re-run the code that's already here, rather than put in this workaround only to take it out later